auth_backends/helsinki_azure_ad.py

```diff
@@ -18,7 +18,11 @@ class HelsinkiAzureADTenantOAuth2(AzureADV2TenantOAuth2):
     name_logged_in_to = pgettext('logged in to []', 'Azure AD')
     name_logout_from = pgettext('log out from []', 'Azure AD')
     name_goto = pgettext('go to []', 'Azure AD')
-    USER_DATA_URL = 'https://graph.microsoft.com/v1.0/me/memberOf?$select=id,displayName,securityEnabled'
+    USER_DATA_URL = (
+            'https://graph.microsoft.com/v1.0/me/memberOf?'
+            '$select=id,displayName,securityEnabled'
+            '$filter=securityEnabled eq true'
```

Comment:
Is this missing `&`? Also [Microsoft Graph Explorer ](https://developer.microsoft.com/en-us/graph/graph-explorer) says the following when adding the `filter` part to the query `The specified filter to the reference property query is currently not supported.`

```suggestion
            '&$filter=securityEnabled eq true'
```